### ISSUE TITLE
AttributeError: module 'comtypes.gen._...' has no attribute 'IEnumVARIANT'

### ISSUE THREAD

**Issue by junkmd** [enthought/comtypes#293]
When I run some code with `comtypes` version 1.1.12, raises `NameError`(#284).

Even if I fixed this error(#287), other error has raised in some modules.

```py
Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import comtypes
>>> comtypes.__version__
'1.1.12'
>>> from comtypes.client import GetModule
>>> GetModule(('{D3295D87-D604-11D4-A704-00C04FA137E4}', 8, 0))  # Microsoft_JScript
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\...\comtypes\comtypes\client\_generate.py", line 159, in GetModule
    mod = _CreateWrapper(tlib, pathname)
  File "C:\Users\...\comtypes\comtypes\client\_generate.py", line 227, in _CreateWrapper
    generate_module(tlib, ofi, pathname)
  File "C:\Users\...\comtypes\comtypes\tools\tlbparser.py", line 767, in generate_module
    gen.generate_code(list(items.values()), filename=pathname)
  File "C:\Users\...\comtypes\comtypes\tools\codegenerator.py", line 290, in generate_code
    self.generate_all(items)
  File "C:\Users\...\comtypes\comtypes\tools\codegenerator.py", line 206, in generate_all
    self.generate(item)
  File "C:\Users\...\comtypes\comtypes\tools\codegenerator.py", line 202, in generate
    mth(item)
  File "C:\Users\...\comtypes\comtypes\tools\codegenerator.py", line 856, in CoClass
    self.generate(itf.get_head())
  File "C:\Users\...\comtypes\comtypes\tools\codegenerator.py", line 202, in generate
    mth(item)
  File "C:\Users\...\comtypes\comtypes\tools\codegenerator.py", line 799, in External
    comtypes.client.GetModule(ext.tlib)
  File "C:\Users\...\comtypes\comtypes\client\_generate.py", line 159, in GetModule
    mod = _CreateWrapper(tlib, pathname)
  File "C:\Users\...\comtypes\comtypes\client\_generate.py", line 242, in _CreateWrapper
    mod = _my_import(fullname)
  File "C:\Users\...\comtypes\comtypes\client\_generate.py", line 26, in _my_import
    return __import__(fullname, globals(), locals(), ['DUMMY'])
  File "C:\Users\...\comtypes\comtypes\gen\_BED7F4EA_1A96_11D2_8F08_00A0C9A6186D_0_2_0.py", line 20960, in <module>
    POINTER(POINTER(comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0.IEnumVARIANT)),
AttributeError: module 'comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0' has no attribute 'IEnumVARIANT'
```

In `comtypes` version 1.1.11, `GetModule(('{D3295D87-D604-11D4-A704-00C04FA137E4}', 8, 0))` was successfully, and `comtypes.gen._00020430_0000_0000_C000_000000000046_0_2_0` has `from comtypes.automation import IEnumVARIANT` line.

IMO `comtypes.tools.codegenerator` has some problems in version 1.1.12.

**Comment by junkmd** [comment]
I PRed #294 for fix this error.

**Comment by junkmd** [comment]
This error was fixed, and released at [1.1.13](https://github.com/enthought/comtypes/releases/tag/1.1.13).
